snmp_juniper/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+  "display_name": "Juniper Networks",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "snmp_juniper",
+  "metric_prefix": "snmp.",
+  "metric_to_check": [
+    "snmp.jnxDcuStatsPackets",
+    "snmp.jnxVirtualChassisPortInPkts",
+    "snmp.jnxScuStatsPackets"
+  ],
+  "creates_events": false,
+  "short_description": "Collect metrics from your Juniper network devices.",
+  "guid": "62f074dc-8b6b-466d-bca2-175194221546",
+  "support": "core",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Juniper Networks",
```

Comment:
I think we should revert this: https://github.com/DataDog/integrations-core/pull/8284/commits/8eff85c5087c94258e66fdd14c624ed44f31adfc

All public_title from other integrations are prefixed with `Datadog-`, not sure where this is used though.

Reply:
This is not used and stripped by https://github.com/DataDog/devtools/blob/master/devtools/devtools/integration/consul_config.py#L26

Reply:
OK, let's keep the prefix (just for being consistent with other integrations)